editor/src/core/shared/multiplayer-hooks.tsx

```diff
@@ -37,3 +55,180 @@ export function useMyUserId(): string | null {
   )
   return myUserId
 }
+
+export function useIsBeingFollowed() {
+  const mode = useEditorState(
+    Substores.restOfEditor,
+    (store) => store.editor.mode,
+    'useIsBeingFollowed mode',
+  )
+
+  const isBeingFollowed = React.useCallback(
+    (connectionId: number) => {
+      return isFollowMode(mode) && mode.connectionId === connectionId
```

Comment:
Does this also need to compare the user ID as well because of the not unique connection IDs?

Reply:
ah yes, I actually chomped it